src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -26,32 +27,78 @@ const getParentIncident = incident => ({
 });
 
 const IncidentSplitContainer = ({ FormComponent }) => {
-  const { data, error, get, isLoading, isSuccess, post } = useFetch();
+  const { error: errorSplit, isSuccess: isSuccessSplit, post } = useFetch();
+  const {
+    data: dataParent,
+    error: errorParent,
+    get: getParent,
+    isLoading: isLoadingParent,
+    isSuccess: isSuccessParent,
+  } = useFetch();
+  const { error: errorUpdate, patch, isSuccess: isSuccessUpdate } = useFetch();
   const { id } = useParams();
   const history = useHistory();
   const dispatch = useDispatch();
-
+  const [parentIncident, setParentIncident] = useState();
+  const [directingDepartment, setDirectingDepartment] = useState([]);
+  const departments = useSelector(makeSelectDepartments);
   const subcategories = useSelector(makeSelectSubCategories);
 
   const subcategoryOptions = useMemo(
     () => subcategories?.map(category => ({ ...category, value: category.extendedName })),
     [subcategories]
   );
 
+  const updateDepartment = useCallback(
+    name => {
+      if (!name || !departments) return;
+
+      const department = departments.list.find(d => d.code === name);
+      setDirectingDepartment(department ? [{ id: department.id }] : []);
+    },
+    [departments, setDirectingDepartment]
+  );
+
   useEffect(() => {
-    get(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`);
-  }, [get, id]);
+    getParent(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`);
+  }, [getParent, id]);
 
   useEffect(() => {
-    if (isSuccess === undefined || error === undefined) return;
+    if (isLoadingParent || isSuccessParent || errorParent === undefined) return;
 
-    const notificationProps = isSuccess
+    if (errorParent === false) {
+      setParentIncident(dataParent);
+    }
+  }, [errorParent, isLoadingParent, isSuccessParent, dataParent]);
+
+  useEffect(() => {
+    if (isSuccessSplit === undefined || errorSplit === undefined) return;
+    if (isSuccessSplit) {
+      patch(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`, { directing_department: directingDepartment });
+    } else {
+      dispatch(
+        showGlobalNotification({
+          title: 'De melding kon niet gesplitst worden',
+          variant: VARIANT_ERROR,
+          type: TYPE_LOCAL,
+        })
+      );
+      history.push(`${INCIDENT_URL}/${id}`);
+    }
+
+    // Disabling linter; patch and history dependencies are generating infinite loops
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [errorSplit, isSuccessSplit, id, dispatch]);
+
+  useEffect(() => {
+    if (isSuccessUpdate === undefined || errorUpdate === undefined) return;
+    const notificationProps = isSuccessUpdate
       ? {
         title: 'De melding is succesvol gesplitst',
         variant: VARIANT_SUCCESS,
       }
       : {
-        title: 'De melding kon niet gesplitst worden',
+        title: 'De melding is gesplits maar het bijwerken van de hoofdmelding regie is niet gelukt',
```

Comment:
This is a bit of a shaky sentence. Apart from the typos, wouldn't it be better to just mention the fact that setting the directing department wasn't successful? The user is directed back to the incident page and it will be visible that the split was successful by the number of child incidents shown there. The error notification should maybe also mention a way to fix the problem.

Reply:
After the discussion with the PO this logic is eliminated for the time being.